report_generator/urls_randomizer.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import logging
+import codecs
+import random
+import traceback
+
+
+"""
+urls_randomizer is used to randomize the urls.txt and turls.txt with a given
+seed.
+
+"""
+
+logging.basicConfig(filename='log/error.log',
+                    level=logging.ERROR,
+                    format='%(asctime)s: %(levelname)s: %(message)s')
+
+
+def main():
+    if len(sys.argv) != 3:
+        logging.critical('Wrong number of arguments.',
+                         'Usage: "python urls_randomizer URLS_FILE SEED"')
+        sys.exit(1)
+    urls_file = sys.argv[1]
+    seed = sys.argv[2]
+    try:
+        seed = int(seed)
+        urls = read_urls(urls_file)
+        save_random_urls(urls, urls_file+'_temp.txt', seed)
+    except ValueError:
```

Comment:
This except clause should happend _immediately_ after the seed = int(seed) line and then continue the normal execution unindented.